gnmi_server/server.go

```diff
@@ -377,6 +386,28 @@ func (s *Server) Get(ctx context.Context, req *gnmipb.GetRequest) (*gnmipb.GetRe
 }
 
 func (s *Server) Set(ctx context.Context, req *gnmipb.SetRequest) (*gnmipb.SetResponse, error) {
+	e := s.ReqFromMaster(req, &s.masterEID)
+	if e != nil {
+		ret, ok := status.FromError(e)
```

Comment:
Why do we these error type checks? Error message can be formatted correctly at the source instead of unboxing and re-boxing here.

Reply:
Done